pallets/committee-management/src/lib.rs

```diff
@@ -249,19 +268,65 @@ pub mod pallet {
 
             Ok(())
         }
+
+        /// Sets ban config, it has an immediate effect
+        #[pallet::call_index(5)]
+        #[pallet::weight((T::BlockWeights::get().max_block, DispatchClass::Operational))]
+        pub fn set_finality_ban_config(
+            origin: OriginFor<T>,
+            minimal_expected_performance: Option<u32>,
+            underperformed_session_count_threshold: Option<u32>,
+            ban_period: Option<EraIndex>,
+        ) -> DispatchResult {
+            ensure_root(origin)?;
+
+            let mut current_committee_ban_config = FinalityBanConfig::<T>::get();
+
+            if let Some(minimal_expected_performance) = minimal_expected_performance {
+                ensure!(
+                    minimal_expected_performance <= 20,
```

Comment:
What is this 20 about?

Reply:
I wanted to have some check on the value and I don't expect that lagging more than 20 rounds will be considered acceptable.

Reply:
Changed to value representing not producing units all session